twoliter/embedded/Makefile.toml

```
@@ -1307,6 +1310,10 @@ if [ "${is_split}" == "yes" ] ; then
    data_volume_args+=(--data-image "${data_image}")
 fi
 
+if [ -f "${BUILDSYS_AMISPEC_TEMPLATE}" ]; then
```

nit: generally `-s` is a better check unless a completely empty file is tolerated:
```suggestion
if [ -s "${BUILDSYS_AMISPEC_TEMPLATE}" ]; then
```